tests/suites/test_suite_psa_crypto.function

```diff
@@ -3101,6 +3101,7 @@ void mac_verify( int key_type_arg,
     psa_algorithm_t alg = alg_arg;
     psa_mac_operation_t operation = PSA_MAC_OPERATION_INIT;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    uint8_t *perturbed_mac = NULL;
```

Comment:
Here again, I think it would be better to add an `expected_return` parameter and to provide the various MAC to verify through the `.data` file.

Reply:
Again, need further discussion, closing this conversation.